ui/apps/platform/cypress/integration/vulnmanagement/reporting/manageReports.test.js

```diff
@@ -2,69 +2,75 @@ import * as api from '../../../constants/apiEndpoints';
-        before(() => {
-            cy.fixture('integrations/notifiers.json').then((response) => {
-                notifiers = response;
-            });
-            cy.fixture('scopes/resourceScopes.json').then((response) => {
-                resourcesScopes = response;
-            });
+        it('should go from left navigation', () => {
+            cy.intercept('GET', api.report.configurations).as('getReportConfigurations');
+            cy.intercept('GET', api.report.configurationsCount).as('getReportConfigurationsCount');
+            visitFromLeftNavExpandable('Vulnerability Management', 'Reporting');
+            cy.wait(['@getReportConfigurations', '@getReportConfigurationsCount']);
+
+            cy.get('h1:contains("Vulnerability reporting")');
+            cy.location('pathname').should('eq', url.reporting.list);
         });
 
-        beforeEach(() => {
-            cy.intercept('GET', api.report.configurations, { reportConfigs: [] }).as(
-                'getReportConfigurations'
-            );
-            cy.intercept('GET', api.report.configurationsCount, { count: 0 }).as(
-                'getReportConfigurationsCount'
+        it('should go to url and select item in nav bar', () => {
+            visitVulnerabilityReporting();
+
+            cy.get(`${navigationSelectors.navExpandable}:contains("Vulnerability Management")`);
+            cy.get(`${navigationSelectors.nestedNavLinks}:contains("Reporting")`).should(
+                'have.class',
+                'pf-m-current'
             );
-            cy.intercept('POST', api.graphql('searchOptions')).as('searchOptions');
-            cy.intercept('GET', api.integrations.notifiers, notifiers).as('getNotifiers');
-            cy.intercept('GET', api.accessScopes.list, resourcesScopes).as('getResourceScopes');
         });
 
-        it('should navigate to the Create Report view by button or directly', () => {
-            cy.visit('/main/dashboard');
-            cy.get(selectors.vulnManagementExpandableNavLink).click({ force: true });
-            cy.get(selectors.vulnManagementExpandedReportingNavLink).click({ force: true });
-            cy.url().should('contain', url.reporting.list);
-
-            cy.wait('@getReportConfigurations');
-            cy.wait('@getReportConfigurationsCount');
-
-            cy.wait('@searchOptions');
-
-            // Hard-coded wait is to ameliorate a tenacious flake in CI that has resisted all more gentle solutions
-            cy.wait(1000);
+        it('should navigate to the Create Report view by button', () => {
+            visitVulnerabilityReporting();
 
+            cy.intercept('GET', api.accessScopes.list).as('getSimpleAccessScopes');
+            cy.intercept('GET', api.integrations.notifiers).as('getNotifiers');
             cy.get(selectors.reportSection.createReportLink).click();
-            cy.location('pathname').should('eq', `${url.reporting.list}`);
+            cy.wait(['@getSimpleAccessScopes', '@getNotifiers']);
```

Comment:
I think Cypress puts server response aliases in the global scope, so we could refactor the `accessScopes` and `notifiers` in the visit-function, and still wait on them after the appropriate action in this and the following test

Reply:
Yeah, although there is a subtle difference which is easy to miss:
* For this test, I removed fixtures to make actual requests.
* For the next test, I kept fixtures because future TODO work will need resource scopes and notifiers.

Reply:
If we want to keep the actual calls for any reasons, then yeah, let's leave them separate. But if we don't care about the actual calls, we could mock it every time.

Or make the helper function take mock names and use conditionally.